draft-kuhn-quic-bdpframe-extension-02.xml

```diff
@@ -496,6 +496,11 @@ BDP_FRAME {
                 	described in Section 18 of 
                 	<xref target="RFC9000"></xref>.
 			</t>
+		        <t> 
+			If the receiver activates the extension, it agrees to receive and read BDP_FRAMEs.
+			If the receiver activates the optimization, it agrees the exploitation of previously computed CC parameters by the sender.
+			The receiver could then agree to do session resumption optimization without actually reading the previous CC parameters.
```

Comment:
/it agrees the exploitation of previously computed CC parameters by the sender./it allows the sender to utilise the previously computed CC parameters.